src/docs/concepts-users-and-authentication/user-permissions/index.md

```diff
@@ -14,35 +14,77 @@ permissions to join, post listings, and start transactions. However, you
 can modify some of these permissions in Console, under the "Access
 control" tab.
```

Comment:
I think here we could have an introductionary paragraph on what this means from a technical perspective => changing these permissions updates the [access-control asset](https://github.com/sharetribe/web-template/commit/e5154d535906ddb4b1d5aabe3a4211be9063e303#diff-2354bc5d624a2c3c0e4a01fa55b9fd8a4e008c0a805a9d7276192ef3f7efce0cR194) and that they are reflected in the currentUser permissions attribute

Reply:
One idea also would explicitly list under each subheading (e.g. Approve users who want to join & Manually grant permission to publish listings) what endpoints this affects (and how) and how the change is reflected in the access-control asset

Reply:
I added a mention and a screenshot of the access-control.json asset now. However, I didn't add endpoint level descriptions at this point, except to clarify the mention in the private marketplace paragraph. My main reason for this decision was that since this is a concepts article and not a reference, we don't need to include all details on how this feature works. If we do want to include a reference article on access control at some point, I think that would be the correct place for endpoint level details, but I would suggest that we only add that after we have the full feature set out (including any Integration API endpoints for managing permissions) since we will be updating this documentation several times during the coming months.